config/backend-config/multitenant-workspace-config.go

```diff
@@ -81,7 +78,7 @@ func (workspaceConfig *MultiTenantWorkspaceConfig) getFromAPI(workspaceArr strin
 
 	backoffWithMaxRetry := backoff.WithMaxRetries(backoff.NewExponentialBackOff(), 3)
 	err := backoff.RetryNotify(operation, backoffWithMaxRetry, func(err error, t time.Duration) {
-		pkgLogger.Errorf("[[ Workspace-config ]] Failed to fetch config from API with error: %v, retrying after %v", err, t)
+		pkgLogger.Errorf("[[MultiTenant-Workspace-config ]] Failed to fetch config from API with error: %v, retrying after %v", err, t)
```

Comment:
>  it because we are missing the logger's name in our log format?
no, we don't. Logger name is there

I don't know why we are doing it like this, cc @cisse21

Reply:
It has been part of legacy code... we can get rid of them

Reply:
removed